src/main/java/game/applicationBase/GameFactory.java

```diff
@@ -0,0 +1,194 @@
+                new HashMap<>(),
+                1);
+
+        /*
+        Scene 2
+         */
+
+        List<String> items2 = new ArrayList<>();
+        items2.add("livingroom-door");
+        items2.add("bike");
+
+        Scene scene2 = new Scene(2, "garage",
+                "A nice place to park your car. But you are too poor and only have an old bike.",
+                items2,
+                new ArrayList<>(),
+                new HashMap<>(),
+                2);
+
+        /*
+        Scene 3
+         */
+
+        Map<String, Integer> nextScenes3 = new HashMap<>();
+        nextScenes3.put("north", 4);
+        Scene scene3 = new Scene(3, "driveway",
+                "The driveway to your garage. It's awfully quiet outside...",
+                new ArrayList<>(),
+                new ArrayList<>(),
+                nextScenes3,
+                3);
+
+        /*
+        Scene 4
+         */
+
+        Map<String, Integer> nextScenes4 = new HashMap<>();
+        nextScenes4.put("south", 3);
+
+        Scene scene4 = new Scene(4, "road",
+                "A nice road, paved last month.",
+                new ArrayList<>(),
+                Arrays.asList("Bully"),
+                nextScenes4,
+                4);
+
+        sceneMap.put(scene0.getId(), scene0);
+        sceneMap.put(scene1.getId(), scene1);
+        sceneMap.put(scene2.getId(), scene2);
+        sceneMap.put(scene3.getId(), scene3);
+        sceneMap.put(scene4.getId(), scene4);
+
+        return sceneMap;
+    }
+    
+    private static Map<String, Character> exampleCharacterMap() {
+        Map<String, Character> characterMap = new HashMap<>();
+
+        Enemy enemy = new Enemy("Bully", "Nasty man that's always out to get you.", 20, new CharacterStats(75, 15, 2));
+
+        List<String> lines = new ArrayList<>();
+        lines.add("This is line one!");
+        lines.add("This is line two!");
+        lines.add("This is my last line! I will repeat it when you talk to me again.");
+
+        Friend friend = new Friend("Brother", "Your younger brother.", 2, new Conversation(lines));
+
+        characterMap.put(enemy.getName(), enemy);
+        characterMap.put(friend.getName(), friend);
+        
+        return characterMap;
+    }
+
+    private static Map<String, Item> exampleItemMap() {
+        Map<String, Item> itemMap = new HashMap<>();
+
+        itemMap.put("cigarettes", new Item("cigarettes", "A pack of luckies, half empty. Or half full?", 10));
+        itemMap.put("joint", new Item("joint", "No idea what's in it", 11));
+        itemMap.put("bedroom-door", new Item("bedroom-door", "A door leading to the livingroom.", 12));
+        itemMap.put("garage-door", new Item("garage-door", "A door leading to the garage.", 13));
+        itemMap.put("livingroom-door", new Item("livingroom-door", "A door leading to the livingroom.", 14));
+        itemMap.put("bike", new Item("bike", "An old bike. It works.", 15));
+
+        return itemMap;
+    }
+
+}
```

Comment:
The code is structured well, it is easy to follow and looks really clean! The comments you guys put in helped a lot too. I can see a design pattern back as well, good job!